Corona-Warn-App/src/main/res/layout/vaccination_consent_fragment.xml

```diff
@@ -41,7 +41,7 @@
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
-                app:srcCompat="@drawable/ic_vaccination_consent" />
+                android:src="@drawable/ic_vaccination_consent" />
```

Comment:
We should not use src ,Please have a look at https://github.com/corona-warn-app/cwa-app-android/pull/2800 ?

Reply:
hm.. this was a fix because images are not displayed in the screenshot when using `app:srcCompat`
The second solution would be to change `ImageView` to `androidx.appcompat.widget.AppCompatImageView` to make it works but that looks ugly. Or do you have any other recommendations?

Reply:
which Android version did you test? We take screenshots on android 29 if that works fine , I would say that is acceptable. Screenshots do not have work on every Android version ,but the App should

Reply:
yes, I tested it with Android API 30 before. I'm reverting the changes.